### PR TITLE
jinja-lsp: init at 0.1.81

### PR DESCRIPTION
**Pull request by Jeidnx**
## Description of changes
Adds jinja-lsp from https://github.com/uros-5/jinja-lsp. I've also added myself as a contributor.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/ji/jinja-lsp/package.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "jinja-lsp";
+  version = "0.1.81";
+
+  src = fetchFromGitHub {
+    owner = "uros-5";
+    repo = "jinja-lsp";
+    rev = "ce972c4bf3fad4da5a91b50610dd40823119ddd0";
```
Comment (dotlambda) [inline pkgs/by-name/ji/jinja-lsp/package.nix:14]:
```suggestion
    rev = "v${version}";
```

File: pkgs/by-name/ji/jinja-lsp/package.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage {
```
Comment (dotlambda) [inline pkgs/by-name/ji/jinja-lsp/package.nix:7]:
```suggestion
rustPlatform.buildRustPackage rec {
```

**Comment by Jeidnx** [comment]
Thanks for your review! I have made some changes, can you take a look again?

File: pkgs/by-name/ji/jinja-lsp/package.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  versionString = "v{$version}";
+in
+rustPlatform.buildRustPackage {
+  inherit version;
+  pname = "jinja-lsp";
+
+  src = fetchFromGitHub {
+    owner = "uros-5";
+    repo = "jinja-lsp";
+    rev = versionString;
```
Comment (dotlambda) [inline pkgs/by-name/ji/jinja-lsp/package.nix:17]:
```suggestion
    rev = "v${version};
```

File: pkgs/by-name/ji/jinja-lsp/package.nix [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    hash = "sha256-6qv1w48tjcBmZzZpeahBsrMgFhKw9sTzvR2A8qeIIGo=";
+  };
+
+  cargoHash = "sha256-F+Gof+muTsQ4V465dW27U51ZsbD7smycY+P2WAeKGLs=";
+
+  meta = {
+    homepage = "https://github.com/uros-5/jinja-lsp";
+    description = "Language Server for jinja";
+    changelog = "https://github.com/uros-5/jinja-lsp/releases/tag/${versionString}";
```
Comment (dotlambda) [inline pkgs/by-name/ji/jinja-lsp/package.nix:26]:
```suggestion
    changelog = "https://github.com/uros-5/jinja-lsp/releases/tag/${src.rev}";
```

**Pull request closed without merging**